examples/semaphore-wf-level.yaml

```diff
@@ -0,0 +1,24 @@
+#apiVersion: v1
+#kind: ConfigMap
+#metadata:
+#  name: my-config
+#data:
+#  workflow: "3"
+#---
```

Comment:
Why is this commented? Also I think we should have documentation/comments explaining the examples.